packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx

```diff
@@ -61,204 +60,184 @@ const CreateDaoBasicInfo: React.FC<{
+          </GridItem>
+          <GridItem gridColumn={'1/2'} gridRow={'1'} alignVertical={'center'}>
+            <StyledText name={'title2'}>Create DAO</StyledText>
+          </GridItem>
+        </Grid>
 
-            <Controller
-              name="daoIdentifier"
-              control={control}
-              defaultValue={daoIdentifier}
-              rules={{ required: 'This is required.' }}
-              render={({ field: { onChange, value }, fieldState: { error } }) => (
-                <TextInput
-                  title="DAO identifier"
-                  subtitle="Enter the indentifier of your DAO (Use only lower capital characters, without space)"
-                  wide
-                  value={value}
-                  placeholder={'Enter DAO identifier'}
-                  onChange={onChange}
-                  status={!!error ? 'error' : 'normal'}
-                  error={error ? error.message : null}
-                />
-              )}
+        <Controller
+          name="daoIdentifier"
+          control={control}
+          defaultValue={daoIdentifier}
+          rules={{ required: 'This is required.' }}
+          render={({ field: { onChange, value }, fieldState: { error } }) => (
+            <TextInput
+              title="DAO identifier"
+              subtitle="Enter the indentifier of your DAO (Use only lower capital characters, without space)"
+              wide
+              value={value}
+              placeholder={'Enter DAO identifier'}
+              onChange={onChange}
+              status={!!error ? 'error' : 'normal'}
+              error={error ? error.message : null}
             />
-            <div>
-              <StyledText name={'title2'}>Create token</StyledText>
-              <StyledText name={'body2'}>
-                Create a new ERC-20 token for your DAO, or use an existing one
-              </StyledText>
-              <Controller
-                name="isExistingToken"
-                control={control}
-                defaultValue={isExistingToken}
-                render={({ field: { onChange, value } }) => (
-                  <ContentSwitcher
-                    onChange={onChange}
-                    selected={value}
-                    items={['New Token', 'Existing Token']}
-                    paddingSettings={{
-                      horizontal: spacing * 2,
-                      vertical: spacing / 4,
-                    }}
-                  />
-                )}
+          )}
+        />
+        <div>
+          <StyledText name={'title2'}>Create token</StyledText>
+          <StyledText name={'body2'}>
+            Create a new ERC-20 token for your DAO, or use an existing one
+          </StyledText>
+          <Controller
+            name="isExistingToken"
+            control={control}
+            defaultValue={isExistingToken}
+            render={({ field: { onChange, value } }) => (
```

Comment:
does the content switch take values 0, 1... instead of boolean? I sometimes see warnings about that.

Reply:
content switcher should accept number or bool, if there is a warning this should be addressed at aragon-ui level